tests/performance/load-tests/README.md

```diff
@@ -1,19 +1,20 @@
 # Overview
-This script tests how well OpenShift environment can handle running simultaneously many of workspaces. It evaluates the performance of the system under test by checking the average results across all pods and identifying failures that occur during the testing process. 
+This script tests the performance of an OpenShift environment by running multiple workspaces simultaneously. It evaluates the system under test by checking the average results across all pods and identifying any failures that occur during the testing process.
```

Comment:
is it OpenShift specific, or would it work on any k8s?

Reply:
I have checked it only on Openshift.

Reply:
ack, it probably not going to work on vanilla k8s, but it is ok for the time-being